src/silx/gui/widgets/WaitingPushButton.py

```diff
@@ -227,6 +227,10 @@ def isWaiting(self):
         :rtype: bool"""
         return self.__waiting
 
+    def close(self):
+        self.stopWaiting()
+        return super().close()
```

Comment:
I needed this to make test pass on ubuntu. Probably safer to have it on the class directly.